web-ui/src/main/resources/catalog/components/common/needhelp/NeedHelpDirective.js

```diff
@@ -61,16 +71,22 @@
           scope.iconOnly = attrs.iconOnly === "true";
           scope.documentationLinks = null;
 
+          scope.helpBaseUrl = "https://docs.geonetwork-opensource.org/latest/{lang}";
```

Comment:
Rather than send this to `latest` which will change over time to 4.6, this should go to `4.4` which is the version that matches this release of geonetwork.  Ideally if we can determine the correct version `4.4`, `4.2`, `3.12` we do not have to modify this code for each branch.

Reply:
I believe it is `{{version}}` / `{{lang}}` as in https://docs.geonetwork-opensource.org/4.2/fr/